libcxx/include/__format/parser_std_format_spec.h

```diff
@@ -593,9 +593,10 @@ class _LIBCPP_TEMPLATE_VIS __parser {
           || (same_as<_CharT, wchar_t> && sizeof(wchar_t) == 2)
 #    endif
   _LIBCPP_HIDE_FROM_ABI constexpr bool __parse_fill_align(_Iterator& __begin, _Iterator __end, bool __use_range_fill) {
-    _LIBCPP_ASSERT_VALID_ELEMENT_ACCESS(__begin != __end,
-                                        "when called with an empty input the function will cause "
-                                        "undefined behavior by evaluating data not in the input");
+    _LIBCPP_ASSERT_VALID_ELEMENT_ACCESS(
+        __begin != __end,
+        "when called with an empty input the function will cause "
```

Comment:
Nit: (hopefully helpful) All other assertion messages begin with capital letters. 
```suggestion
        "When called with an empty input, the function will cause "
```

Reply:
If you think that this suggestion is helpful, I can put them on the others to make it easier for you.

Reply:
Thanks! It's a good suggestion, but for this PR specifically, I'd prefer to keep the delta as small as possible, even if it means leaving some less-than-perfect error messages in place. It would be good to do a follow-up to improve some messages, though.